### ISSUE TITLE
Comparator Not Working As Intended For Test 3 / 6

### ISSUE THREAD

**Issue by NingXinran** [nus-cs2030/2223-s2#235]
# Summary

Hi, i wrote a comparator for level5.jsh that compares first by fare then by waiting time. It works for the first 2 test cases and for tests i wrote on my own, but somehow gives this strange order for test 3. Anyone has any idea why?

## Screenshots
![Screenshot 2023-02-23 at 7 25 16 PM](https://user-images.githubusercontent.com/70146427/220892998-52c46eac-5fb1-464e-96be-34fed4a4a5d5.png)

**Comment by coderlimjingcheng** [comment]
Hi, I encountered a similar problem. Apparently, test3.jsh will test your comparator class at level 3. So, if you have implemented another comparator class for level5.jsh, then the problem probably lies in the first comparator class that you have built. For example, in my level 3 of this problem, I used a comparator class called CompareBooking.java. When I was at level 5, I decided that the CompareBooking.java is irrelevant, so I went on and implemented another comparator class called CompareBooking2.java. So, I had to fix my CompareBooking.java to pass this test case, even if my CompareBooking2.java works perfectly.
